share/spice/forecast/forecast.css

```diff
@@ -274,3 +272,20 @@
 #fe_temp_switch .is-active {
 	font-weight: bold;
 }
+
+/*
+    Dark mode specific changes
+*/
+
+.dark-bg .fe_label a:hover {
+    color: #dadada;
+    text-decoration: underline;
+}
+
+.dark-bg a#fe_temp_switch:hover {
+    color: #c2c2c2;
+}
+
+.dark-bg .fe_icon { display: none; }
```

Comment:
This doesn't seem necessary? I always see the correct icons in Chrome/FF/Safari. Which browser are you using?